.github/workflows/bypass-version-review.yaml

```diff
@@ -0,0 +1,34 @@
+name: Bypass Version File Review by Admin
```

Comment:
Do we need this workflow anymore? It looks like this case is now handled by `bypass-review.yaml`

Reply:
removed